src/Microsoft.Azure.Functions.Extensions.Mcp/Backplane/BackplaneService.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol.Protocol.Messages;
+using System.Threading.Channels;
+
+namespace Microsoft.Azure.Functions.Extensions.Mcp.Backplane
+{
+    internal sealed class BackplaneService(IMcpClientSessionManager sessionManager, IMcpBackplane backplane, IMcpInstanceIdProvider instanceIdProvider, ILogger<BackplaneService> logger) : IHostedService, IMcpBackplaneService
```

Comment:
Do we want this to be an IDisposable as well for any cleanup we might need?

Reply:
Are you referring to the cleanup of the `_backplaneProcessingTask` instance? 

That task is cached and returned the the caller in `StopAsync`, so the expected flow here matching the hosted service lifecycle wouldn't leave a running, undisposed Task behind.

We could consider checking in scenarios where this is not called, but we shouldn't expect DI to call dispose on this type without stopping the service, which allows for that cleanup.

I did however find an urelated problem when looking at `StartAsync`. That shouldn't return the cached `Task` and this was only working because the service was never registered as a hosted service. I'm fixing both of those problems.